schema/crdb/38.0.0/up01.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS omicron.public.inv_physical_disk (
```

Comment:
Can/should we include the slot number here?  (if so, can/should we use that for pagination/uniqueness?)

Reply:
Sure, we'll probably want that out of inventory anyway. Also, I agree, this seems like a better PK ("sled id" + "slot") than the reported values of serial/model/vendor.

Updated to plumb through the slot from the sled agent, and using as pagination key in this PR.